docs/docsite/rst/roadmap/COLLECTIONS_3_0.rst

```diff
@@ -0,0 +1,45 @@
+===================
+Ansible project 3.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_11` for the most recent updates on ``ansible-base``.
+
+.. contents::
+   :local:
+
+Release schedule
+=================
+
+.. note::
+
+    Ansible its switching from its traditional versioning scheme to `semantic versioning <https://semver.org/>`_ for this release.  So the version will be 3.0.0 instead of 2.11.0.
```

Comment:
```suggestion
    Ansible is switching from its traditional versioning scheme to `semantic versioning <https://semver.org/>`_ for this release.  So this version is 3.0.0 instead of 2.11.0.
```